tests/Magick.NET.Tests/MagickImageTests/TheShaveMethod.cs

```diff
@@ -10,6 +11,30 @@ public partial class MagickImageTests
 {
     public class TheShaveMethod
     {
+        [Fact]
+        public void ShouldThrowExceptionWhenSizeIsNegative()
+        {
+            using var image = new MagickImage(Files.Builtin.Logo);
```

Comment:
Don't think we need to load an image here.

Reply:
Duly noted. So your commits removed image loading. Will update this PR.